src/client/opamSolution.ml

```diff
@@ -138,6 +138,36 @@ let atom_of_name name =
   name, None
 *)
 
+let did_you_mean st atoms = 
```

Comment:
This ended up here instead of `OpamClient` as suggested, since it would create a cycle @kit-ty-kate
So the question stays open (just for info, moving `check_availability` and `sanitize_atom_list` somewhere higher would work. Otherwise its an another module.)

Reply:
Done.